meson.build

```diff
@@ -715,8 +718,11 @@ if id_result.returncode() == 0
                 message('WARNING:\n' +
                         '        The local user with the configured user name "@0@" of the nobody user does not have UID 65534 (it has @1@).\n'.format(nobody_user, id) +
                         '        Your build will result in an user table setup that is incompatible with the local system.')
+                nobody_group_mismatch = true
```

Comment:
This doesn't seem to be correct. `id -u` above checks whether `nobody_user` has a certain id, so it has nothing to do with groups. The tests are likely to still be failing.

Reply:
```sh
$ meson -Dnobody-user=nobody -Dnobody-group=nobody build && ninja -C ./build
...
Message: WARNING:
        The local user with the configured user name "nobody" of the nobody user does not have UID 65534 (it has 99).
        Your build will result in an user table setup that is incompatible with the local system.
Message: WARNING:
        The local group with the configured group name "nobody" of the nobody group does not have UID 65534 (it has 99).
        Your build will result in an group table setup that is incompatible with the local system.
...

$ grep -r MISMATCH build/config.h
#define NOBODY_GROUP_MISMATCH 1
#define NOBODY_USER_MISMATCH 0

$ sudo touch /etc/systemd/dont-synthesize-nobody

$ sudo ./build/test-user-util
/* test_uid_to_name_one(0, "root") */
/* test_uid_to_name_one(65534, "nobody") */
/* test_uid_to_name_one(65535, "65535") */
/* test_uid_to_name_one(4294967295, "4294967295") */
/* test_gid_to_name_one(0, "root") */
/* test_gid_to_name_one(65534, "nobody") */
/* test_gid_to_name_one(5, "tty") */
/* test_gid_to_name_one(65535, "65535") */
/* test_gid_to_name_one(4294967295, "4294967295") */
/* test_get_user_creds_one("root", "root", 0, 0, "/root", "/bin/sh") */
got "root", 0, 0, "/root", "/bin/sh"
/* test_get_user_creds_one("0", "root", 0, 0, "/root", "/bin/sh") */
got "root", 0, 0, "/root", "/bin/sh"
/* test_get_user_creds_one("nobody", "nobody", 65534, 65534, "/", "/sbin/nologin") */
got "nobody", 99, 99, "/", "/sbin/nologin"
Assertion 'ruid == uid' failed at ../src/test/test-user-util.c:181, function test_get_user_creds_one(). Aborting.
Aborted
```

Reply:
I don't see how that can be. You get two warnings, but only NOBODY_GROUP_MISMATCH is set. Did you paste from the wrong build?